private/model/api/api.go

```diff
@@ -326,6 +332,71 @@ var tplAPI = template.Must(template.New("api").Parse(`
 {{- end }}
 `))
 
+var tplUnmodelledAwsQueryCompatible = template.Must(
+	template.New("tplUnmodelledAwsQueryCompatible").Parse(`
+	{{- if $.AwsQueryCompatible }}
+		type UnmodelledException struct {
```

Comment:
For now we probably want to keep this type unexported, i.e. `unmodeledAWSQueryException`

Minor nit we use the US spelling of modeled :)